question-servers/freeformPythonLib/prairielearn.py

```diff
@@ -810,6 +810,42 @@ def matlab_to_numpy(a):
             return (None, 'Invalid format (missing square brackets and not a real number).')
 
 
+def latex_array_from_numpy_array(A, presentation_type='f', digits=2):
+
+    """latex_array_from_numpy_array
+    This function assumes that A is one of these things:
+            - a number (float or complex)
+            - a 2D ndarray (float or complex)
+    It returns A as a Latex-formatted matrix
```

Comment:
It might be helpful to clarify the meaning of "Latex-formatted matrix" - for example, saying that this function returns a string with the form `\begin{bmatrix} ... & ... \\ ... & ... \end{bmatrix}`.

Maybe also say what the arguments `presentation_type` and `digits` do?